src/fvtt/page-script.js

```diff
@@ -20,13 +20,22 @@ class FVTTDisplayer {
                 buttons[button]();
             });
         });
-        return this._postChatMessage(html, character, whisper, play_sound);
+        return this._postChatMessage(html, character, whisper, play_sound, attack_rolls, damage_rolls);
     }
 
-    _postChatMessage(message, character, whisper, play_sound = false) {
+    _postChatMessage(message, character, whisper, play_sound = false, attack_rolls, damage_rolls) {
         const MESSAGE_TYPES = CONST.CHAT_MESSAGE_TYPES || CHAT_MESSAGE_TYPES;
+        const pool = new DicePool([...attack_rolls, ...damage_rolls.map(d => d[1])].map(r => r._roll));
+        pool.roll();
+        const pool_roll = new Roll("");
```

Comment:
Would be nice to make the formula here, like (to be tested) :
```js
const formulas = pool.dice.map(d => d.formula);
new Roll(`{${formulas.join(",")}}`);
```

Reply:
Added as requested